### ISSUE TITLE
As a user, I want the app to show the accessibility information for each place (e.g., wheelchair access, entrance for strollers), to ensure it is suitable for all users.

### ISSUE THREAD

**Issue by Saponzo** [celiajosso/erasmus-sds#6]
Acceptance Criteria:

-     Information about accessibility should be provided for each place.
-     It should mention whether a place is wheelchair accessible, has strollers, etc.

**Comment by Saponzo** [comment]
Planning: 2
